nautobot/extras/models/mixins.py

```diff
@@ -18,3 +25,68 @@ def dynamic_groups(self):
             self._dynamic_group_queryset = queryset
 
         return self._dynamic_group_queryset
+
+
+# 2.0 TODO: Remove after v2 since we will no longer care about backwards-incompatibility.
+# - Remove the `monkey_mix` from `nautobot.core.apps.CoreConfig.ready()`
+# - Convert this into a subclass instead of a mixin and move it to `nautobot.extras.models.managers.TaggableManager` (assuming this stays in `extras`)
+# - Replace all `from taggit.managers import TaggableManager` references to `from nautobot.extras.models.managers import TaggableManager`
```

Comment:
I'm thinking we might want to go ahead and track this as a housekeeping item for v2 if we haven't already.